internal/security/grantvmgroupaccess.go

```diff
@@ -68,11 +73,11 @@ const (
 // include Grant ACE entries for the VM Group SID. This is a golang re-
 // implementation of the same function in vmcompute, just not exported in
 // RS5. Which kind of sucks. Sucks a lot :/
-func GrantVmGroupAccess(name string) error {
+func GrantVmGroupAccess(name string, desiredAccess AccessMask) error {
 	// Stat (to determine if `name` is a directory).
 	s, err := os.Stat(name)
 	if err != nil {
-		return errors.Wrapf(err, "%s os.Stat %s", gvmga, name)
+		return fmt.Errorf("%s os.Stat %s: %w", gvmga, name, err)
```

Comment:
My golang is fading lol. Why `%w` and not `%v` for this case?

Reply:
`%w` enables error unwrapping https://pkg.go.dev/fmt@go1.18#Errorf

Reply:
Lolz. I remembered this exactly opposite for unwrap. Thanks